packages/core/src/controllers/relayer.ts

```diff
@@ -84,6 +88,9 @@ export class Relayer extends IRelayer {
     }
   >();
 
+  private pingTimeout: NodeJS.Timeout | undefined;
+  private heartBeatTimeout = toMiliseconds(THIRTY_SECONDS + ONE_SECOND);
```

Comment:
Where do these numbers come from? Can we add a comment here

Reply:
Discussion with relay team. The relay pings 30s after no messages were transmimitted so 31s is safe to assume connection is dead

Reply:
Ack on context, can we add as comment pls as suggested 🙏 They happen to be constants here but it's still effectively a magic number like this :)